komga-webui/src/locales/en.json

```diff
@@ -342,21 +342,21 @@
       "field_import_barcode_isbn": "ISBN barcode",
       "field_import_comicinfo_book": "Book metadata",
       "field_import_comicinfo_collections": "Collections",
-      "field_import_comicinfo_readlists": "Read lists",
+      "field_import_comicinfo_readlists": "Reading lists",
       "field_import_comicinfo_series": "Series metadata",
       "field_import_epub_book": "Book metadata",
       "field_import_epub_series": "Series metadata",
       "field_import_local_artwork": "Local artwork",
       "field_import_mylar_series": "Series metadata",
       "field_name": "Name",
-      "field_repair_extensions": "Automatically repair incorrect file extensions",
+      "field_repair_extensions": "Auto-repair incorrect file extensions",
       "field_root_folder": "Root folder",
       "field_scanner_deep_scan": "Deep scan",
       "field_scanner_empty_trash_after_scan": "Empty trash automatically after every scan",
       "field_scanner_force_directory_modified_time": "Force directory modified time",
       "field_series_cover": "Series cover",
       "file_browser_dialog_button_confirm": "Choose",
-      "file_browser_dialog_title": "Library's root folder",
```

Comment:
Redundant

Reply:
Redundant with?

Reply:
Redundant to the alternative

Reply:
> Redundant to the alternative

i don't understand what you are saying

Reply:
"'s" can be omitted, and was done away with for that reason.